samples/IstioFilterPattern.yaml

```
@@ -1,8 +1,10 @@
+#Pre-requisite: A Kubernetes cluster  with a running Istio control plane
+#Description: This Patternfile injects rate limiter filter in the dataplane in istio-test namespace
```

```suggestion
#Description: This design injects the rate limiter filter into the dataplane of the `default` namespace
```

Wording change (no more "patternfile") and filters should not be injected into the Istio control plane, but into workload-bearing namespaces.